### ISSUE TITLE
cutting power early in the first boot can result in a device that can't provision

### ISSUE THREAD

**Issue by shaunmulligan** [resin-os/resinos#311]
**Tested on:**
- RPI3
- resinOS 2.0.3+rev1.dev

** How to reproduce:**
- boot up a fresh OS image, with hdmi plugged in
- pull out the usb power ~3 seconds after the resin.io splash logo appears on screen.
- now reboot the device ( 2/3 times I did this, the device was unable to connect to the VPN and wasn't able to run the supervisor.

**Some additional logs:**
Supervisor Logs
```
May 18 19:06:04 f6fe6a1 docker[1212]: Failed to stop container (resin_supervisor): Error response from daemon: No such container: resin_supervisor
May 18 19:06:04 f6fe6a1 systemd[1]: Started Resin supervisor.
May 18 19:06:04 f6fe6a1 start-resin-supervisor[1221]: docker: "inspect" requires a minimum of 1 argument.
May 18 19:06:04 f6fe6a1 start-resin-supervisor[1221]: See 'docker inspect --help'.
May 18 19:06:04 f6fe6a1 start-resin-supervisor[1221]: Usage:        docker inspect [OPTIONS] CONTAINER|IMAGE [CONTAINER|IMAGE...]
May 18 19:06:04 f6fe6a1 start-resin-supervisor[1221]: Return low-level information on a container or image
May 18 19:06:04 f6fe6a1 systemd[1]: resin-supervisor.service: Main process exited, code=exited, status=1/FAILURE
May 18 19:06:04 f6fe6a1 docker[1252]: Failed to stop container (resin_supervisor): Error response from daemon: No such container: resin_supervisor
May 18 19:06:04 f6fe6a1 systemd[1]: resin-supervisor.service: Unit entered failed state.
May 18 19:06:04 f6fe6a1 systemd[1]: resin-supervisor.service: Failed with result 'exit-code'.
May 18 19:06:14 f6fe6a1 systemd[1]: resin-supervisor.service: Service hold-off time over, scheduling restart.
May 18 19:06:14 f6fe6a1 systemd[1]: Stopped Resin supervisor.
May 18 19:06:14 f6fe6a1 systemd[1]: Starting Resin supervisor...
May 18 19:06:15 f6fe6a1 docker[1265]: Failed to stop container (resin_supervisor): Error response from daemon: No such container: resin_supervisor
May 18 19:06:15 f6fe6a1 systemd[1]: Started Resin supervisor.
May 18 19:06:15 f6fe6a1 start-resin-supervisor[1275]: docker: "inspect" requires a minimum of 1 argument.
May 18 19:06:15 f6fe6a1 start-resin-supervisor[1275]: See 'docker inspect --help'.
May 18 19:06:15 f6fe6a1 start-resin-supervisor[1275]: Usage:        docker inspect [OPTIONS] CONTAINER|IMAGE [CONTAINER|IMAGE...]
May 18 19:06:15 f6fe6a1 start-resin-supervisor[1275]: Return low-level information on a container or image
May 18 19:06:15 f6fe6a1 systemd[1]: resin-supervisor.service: Main process exited, code=exited, status=1/FAILURE
May 18 19:06:15 f6fe6a1 docker[1305]: Failed to stop container (resin_supervisor): Error response from daemon: No such container: resin_supervisor
May 18 19:06:15 f6fe6a1 systemd[1]: resin-supervisor.service: Unit entered failed state.
May 18 19:06:15 f6fe6a1 systemd[1]: resin-supervisor.service: Failed with result 'exit-code'.
```
openvpn:
```
root@033c8f6:~# journalctl -u openvpn-resin |tail
May 18 21:46:30 033c8f6 openvpn[846]: Data Channel Encrypt: Cipher 'BF-CBC' initialized with 128 bit key
May 18 21:46:30 033c8f6 openvpn[846]: Data Channel Encrypt: Using 160 bit message hash 'SHA1' for HMAC authentication
May 18 21:46:30 033c8f6 openvpn[846]: Data Channel Decrypt: Cipher 'BF-CBC' initialized with 128 bit key
May 18 21:46:30 033c8f6 openvpn[846]: Data Channel Decrypt: Using 160 bit message hash 'SHA1' for HMAC authentication
May 18 21:46:30 033c8f6 openvpn[846]: Control Channel: TLSv1, cipher TLSv1/SSLv3 DHE-RSA-AES256-SHA, 2048 bit RSA
May 18 21:46:30 033c8f6 openvpn[846]: [server] Peer Connection Initiated with [AF_INET]52.5.48.142:443
May 18 21:46:32 033c8f6 openvpn[846]: SENT CONTROL [server]: 'PUSH_REQUEST' (status=1)
May 18 21:46:32 033c8f6 openvpn[846]: AUTH: Received control message: AUTH_FAILED
May 18 21:46:32 033c8f6 openvpn[846]: SIGUSR1[soft,auth-failure] received, process restarting
May 18 21:46:32 033c8f6 openvpn[846]: Restart pause, 10 second(s)
```

**Comment by shaunmulligan** [comment]
I deleted the contents of `/var/lib/docker` and rebooted the device and it seems to provision and started downloading a container.

**Comment by petrosagg** [comment]
@agherzan what are we doing with this issue?

**Comment by agherzan** [comment]
This should be fixed since 2.1.0.
